examples/jupyter_lab.yaml

```diff
@@ -0,0 +1,22 @@
+# Example: Launch Jupyter Lab and auto-expose its port to Internet.
+#
+# Usage:
+#     $ sky launch -c jupyter jupyter_lab.yaml
+#     # Then look for the logs for some output like:
+#     # Jupyter Server 2.7.0 is running at:
+#     # http://ip-x-x-x-x:29324/lab?token=<token>
```

Comment:
This prefix depends on what cloud we are using. Maybe only keep the next line is fine.